packages/webapp/src/config.ts

```diff
@@ -1,3 +1,11 @@
+// dayjs setup
+import dayjs from "dayjs";
+import * as localizedFormat from "dayjs/plugin/localizedFormat";
+import * as relativeTime from "dayjs/plugin/relativeTime";
+dayjs.extend(localizedFormat.default);
+dayjs.extend(relativeTime.default);
+// end dayjs setup
```

Comment:
I think I mentioned this on the config.ts file but since it's just app setup I wonder if it should stay at the `src/_app.tsx`. Thoughts?

Reply:
That's where I went first...and then I read your original comment. haha! I agree with you; I'll put it in `_app.tsx`. 👍